### ISSUE TITLE
Author Metadata: Shashank Gupta

### ISSUE THREAD

**Issue by shatu** [acl-org/acl-anthology#3658]
### Author Pages

https://aclanthology.org/people/s/shashank-gupta/

### Author ORCID

https://orcid.org/0000-0002-3683-3739

### Institution of highest (anticipated) degree

University of Illinois at Urbana-Champaign, IL, US

### Your papers (if required, see comment below)

-Provide Anthology IDs or Anthology URLs here-

### Type of Author Metadata Correction

- [X] The author page wrongly conflates different people with the same name.
- [ ] This author has multiple pages with different spellings of their name.
- [ ] This author has permanently changed their name.

### Supporting Information

Only the following paper from the current list belongs to me:
- L18-1086

My academic profile: [Semantic Scholar](https://www.semanticscholar.org/author/Shashank-Gupta/2152953535), [Personal Webpage](https://shashankgupta.info/)

The other authors being conflated appear to be from [UT Austin](https://scholar.google.com/citations?user=UYT2VwkAAAAJ&hl=en), [University of Amsterdam](https://scholar.google.co.in/citations?user=UvTcU-IAAAAJ&hl=en), and [Amazon AWS](https://www.linkedin.com/in/shashank-gupta-0b34ab2a/?originalSubdomain=in)

**Comment by mjpost** [comment]
Hello: we are attempting to close out a large backlog of author page requests. As part of these efforts, we are collecting additional information ([ORCID](https://orcid.org/) and degree institution) which will help us assign papers to the correct author in the future. Please modify the updated description above with the requested information.

If you are requesting to split an author page (i.e., your page has some papers that are not yours), please also provide a list of your papers, in the form of Anthology IDs or URLS (e.g., 2023.wmt-1.13 or https://aclanthology.org/2023.wmt-1.13/). We are unable to match papers to Google or Semantic Scholar profiles.

**Comment by shatu** [comment]
Thanks; Done

**Comment by shatu** [comment]
Hi @mjpost, 

It seems that some of the entries have been disambiguated incorrectly. I'm the same person as the ["Shashank Gupta"](https://shashankgupta.info/) from "Allen Institute for Artificial Intelligence", and the following papers in the ACL Anthology were co-authored by me:

1. [AppWorld: A Controllable World of Apps and People for Benchmarking Interactive Coding Agents](https://aclanthology.org/2024.acl-long.850/)
2. [SUPER: Evaluating Agents on Setting Up and Executing Tasks from Research Repositories](https://aclanthology.org/2024.emnlp-main.702/)

I have updated my ORCID record to reflect this: https://orcid.org/0000-0002-3683-3739

**Comment by mjpost** [comment]
ping @Azax4

**Comment by mjpost** [comment]
Your original post mentioned just one paper. I assume these others were published in the intervening time. We'll get this corrected in short order.

**Comment by Azax4** [comment]
Closed by #5720

**Comment by mjpost** [comment]
Hi @shatu: #5720 is our set of monthly corrections. We'll merge these soon after the turn of the month.

**Comment by shatu** [comment]
@mjpost, @Azax4: Thank you! 
Any chance I can quickly review what my author page will look like after the fixes to make sure that we don't need any additional corrections?

**Comment by mjpost** [comment]
Yes, navigate to #5720, then to the preview comment, then to your author page underneath: https://preview.aclanthology.org/corrections-2025-08/people/shashank-gupta/

**Comment by shatu** [comment]
Thanks @mjpost. 
One last question: I noticed my profile lists my affiliation as UIUC, instead of “Allen Institute for Artificial Intelligence (Ai2).” I was at UIUC from 2015–2017, but Ai2 has been my affiliation for the past three years.

What’s the policy on which affiliation is shown—most recent school attended or most recent/current institution?

**Comment by mjpost** [comment]
The "affiliation" is where you had obtained your highest degree when first publishing. We should clarify this. We don't have the bandwidth to maintain current affiliations.

**Comment by shatu** [comment]
Sounds good, and makes sense 👍🏻
